docs/knative-offerings.md

```diff
@@ -18,6 +18,7 @@ Here is a list of commercial Knative products (alphabetically):
 - [Managed Knative for IBM Cloud Kubernetes Service](https://cloud.ibm.com/docs/containers?topic=containers-serverless-apps-knative): is a managed add-on for the [IBM Kubernetes Service](https://cloud.ibm.com/docs/containers?topic=containers-getting-started) that enables you to deploy and manage Knative based services on your own Kubernetes cluster.
 - [Openshift Serverless](https://access.redhat.com/documentation/en-us/openshift_container_platform/4.3/html-single/serverless_applications/index): on OpenShift Container Platform enables stateful, stateless, and serverless workloads to all run on a single multi-cloud container platform with automated operations. Developers can use a single platform for hosting their microservices, legacy, and serverless applications.
 - [Pivotal Function Service (PFS)](https://docs.pivotal.io/pfs): is a platform for building and running functions, applications, and containers on Kubernetes. PFS is based on the [riff](https://projectriff.io) open source project.
```

Comment:
Hi @sebgoa :slightly_smiling_face: 
One minor nit is that this addition is in a slightly different format to the others, e.g:

`: is a platform...[]` vs `: A fully-managed`
I would either update the other entries to say ": A platform...[]" or I would update the TriggerMesh entry to the format of the others, e.g. ": is a fully managed...[]".
Also, I'm not sure about the use of 'fully-managed' vs 'fully managed', but this is probably something to refer to any contributors guidelines for.

Reply:
What about the openshift entry ? ": on Openshift container.."

Reply:
I'd change it to: 

: enables stateful, stateless, and serverless workloads to run on OpenShift Container Platform with automated operations. Developers can use a single platform for hosting their microservices, legacy, and serverless applications.

Thanks :slightly_smiling_face:

Reply:
@bbrowning can you actually check that me moving this sentence around doesn't wreck it cos I think you added this. Basically we want these lines to be formatted the same way but maybe we can just move "with automated operations" too, because I actually find that confusing and I'm not sure it adds anything here?

Reply:
changing this openshift specific verbiage should be done in a separate PR by an openshift person.

Reply:
@sebgoa that's OK, I can do it, was just trying to save adding another PR for review since it's a small fix.